scripts/docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+name: uaa
+
+services:
+  postgres:
+    image: "postgres:17.2"
```

Comment:
please check if https://github.com/cloudfoundry/uaa/blob/develop/scripts/start_db_helper.sh can be re-used

Reply:
PR still in Work in Progress mode

We wrote this on MacOS, I see failures on Linux. Working on those today